purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -47,28 +53,29 @@ data class PaywallData(
         /**
          * Whether the background image will be blurred (in templates with one).
          */
-        val blurredBackgroundImage: Boolean = false,
+        @SerialName("blurred_background_image") val blurredBackgroundImage: Boolean = false,
 
         /**
          * Whether a restore purchases button should be displayed.
          */
-        val displayRestorePurchases: Boolean = true,
+        @SerialName("display_restore_purchases") val displayRestorePurchases: Boolean = true,
 
         /**
          * If set, the paywall will display a terms of service link.
          */
-        val termsOfServiceURL: URL? = null,
+        @SerialName("tos_url") @Serializable(with = URLSerializer::class) val termsOfServiceURL: URL? = null,
 
         /**
          * If set, the paywall will display a privacy policy link.
          */
-        val privacyURL: URL? = null,
+        @SerialName("privacy_url") @Serializable(with = URLSerializer::class) val privacyURL: URL? = null,
```

Comment:
Does this ignore errors if the URL is invalid? Better for it to become `null` than failing the decode the whole thing

Reply:
I haven't actually tested it. I'll check but I think it's going to crash. Good call